roles/edpm_telemetry/defaults/main.yml

```diff
@@ -21,7 +21,7 @@ edpm_telemetry_config_src: /var/lib/openstack/configs/telemetry
 # Directory in the compute node
 edpm_ceilometer_config_dest: /var/lib/openstack/config
 # Image to use for node_exporter
-telemetry_node_exporter_image: quay.io/prometheus/node-exporter:v1.5.0
+edpm_node_exporter_image: quay.io/prometheus/node-exporter:v1.5.0
```

Comment:
Since you're changing the name, can we go ahead and change it to match our guidelines? See docs/source/contributing_roles.rst. It should be `edpm_telemetry_node_exporter_image`.

Reply:
Done, that also requires that the default variables in the dataplane-operator meet that same guidelines. That has been pushed in this PR: https://github.com/openstack-k8s-operators/dataplane-operator/pull/722